### PR TITLE
winport: update main window title with current game

### PR DESCRIPTION
**Pull request by MatrixDJ96**
Fixed previous pull request

### CODE REVIEW

File: desmume/src/frontend/windows/main.cpp [hunk 1]
```diff
@@ -2961,6 +2961,46 @@ void WavEnd()
...
+		if (curr_title == nullptr) {
+			curr_title = EMU_DESMUME_NAME_AND_VERSION();
+		}
+
+		char new_title[512];
+		char game_title[128];
+
+		strcpy(new_title, curr_title);
+
+		int new_lenght = strlen(new_title);
```
Comment (Jules-A) [inline desmume/src/frontend/windows/main.cpp:2977]: Hate to be that guy but you spelt "length" wrong. Oh and while this file is a mess, variables are mostly camelCase. I'm not a maintainer so you can ignore what I said but it kind of triggered me.

**Comment by zeromus** [comment]
looks fine to me. but since jules pointed it out, yeah, please fix typo. I dont care about code style for something as small as variable names internal to function

File: desmume/src/frontend/windows/main.cpp (hunk 1 above)
Comment (MatrixDJ96) [reply desmume/src/frontend/windows/main.cpp:2977]: Sorry, fixed

**Comment by MatrixDJ96** [comment]
@zeromus is now ok?

**Pull request merged**